pkgs/misc/lssecret/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, pkg-config
+, libsecret
+, fetchFromGitLab
+, lib
+}:
```

```suggestion
{ lib
, stdenv
, fetchFromGitLab
, pkg-config
, libsecret
}:
```

nit: ordering, start with lib, stdenv, and non-packages